cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -165,7 +165,6 @@ input {
 
         &:focus {
             box-shadow: 0 0 0 1px @pacific inset;
-            outline: none;
```

Comment:
I wonder what this was used for originally. Looks like this change will affect the global search too, but it looks fine as far as I can tell.